src/main/scala/fable/Consumer.scala

```diff
@@ -175,6 +175,25 @@ class Consumer[F[_]: ContextShift: Monad: Sync, K, V] private[fable] (
       }.toMap
     }
 
+  /**
+    * Get metadata about partitions for all topics that the user is authorized to view. This method will issue a remote call to the server.
```

Comment:
I think this is longer than 80 characters?

Reply:
I did run scalafmt, does that not run on comments?

Reply:
I think it doesn't wrap comments for some reason, unfortunately.